biogeochem/EDPhysiologyMod.F90

```diff
@@ -1843,33 +1853,46 @@ subroutine SeedIn( currentSite, bc_in )
           do pft = 1,numpft
 
              if(currentSite%use_this_pft(pft).eq.itrue)then
-                ! Seed input from local sources (within site)
-                litt%seed_in_local(pft) = litt%seed_in_local(pft) + site_seed_rain(pft)/area
-
-                ! If there is forced external seed rain, we calculate the input mass flux
-                ! from the different elements, using the mean stoichiometry of new
-                ! recruits for the current patch and lowest canopy position
-
-                select case(element_id)
-                case(carbon12_element)
-                   seed_stoich = 1._r8
-                case(nitrogen_element)
-                   seed_stoich = currentPatch%nitr_repro_stoich(pft)
-                case(phosphorus_element)
-                   seed_stoich = currentPatch%phos_repro_stoich(pft)
-                case default
-                   write(fates_log(), *) 'undefined element specified'
-                   write(fates_log(), *) 'while defining forced external seed mass flux'
-                   call endrun(msg=errMsg(sourcefile, __LINE__))
-                end select
+             
+             ! Seed input from local sources (within site)
+             litt%seed_in_local(pft) = litt%seed_in_local(pft) + site_seed_rain(pft)/area
 
-                ! Seed input from external sources (user param seed rain, or dispersal model)
-                seed_in_external =  seed_stoich*EDPftvarcon_inst%seed_suppl(pft)*years_per_day
-                litt%seed_in_extern(pft) = litt%seed_in_extern(pft) + seed_in_external
+             ! If we are using the Tree Recruitment Scheme (TRS) with or w/o seedling dynamics
+             if ( any(regeneration_model == [TRS, TRS_no_seedling_dyn]) .and. &
+                  prt_params%allom_dbh_maxheight(pft) > min_max_dbh_for_trees) then
+             
+             ! Send a fraction of reproductive carbon to litter to account for 
+             ! non-seed reproductive carbon (e.g. flowers, fruit, etc.)
```

Comment:
@adamhb I was expecting %seed_in_local to be reduced per the same amount that was sent to decay, but I can't find that reduction. Am I interpreting this wrong?

Reply:
nevermind, they are both fluxes applied simultaneously